tests/darwin/torch/dataset_test.py

```diff
@@ -4,8 +4,10 @@
 from unittest.mock import patch
 
 import numpy as np
+import pytest
```

Comment:
Unclear how these worked without this tbh!  I guess they only worked in a pytest session where it's already a global.

Reply:
Yeah I was confused too.

Reply:
Global scope I guess